### DISCUSSION TITLE
E-commerce Framework

### DISCUSSION THREAD

**Discussion by designjpp** [BrandOriented/PimcoreCiHubConnector#67] [General]
Hello, 

Hello, 

I tried to install the Ci-Hub integration, but Elasticsearch didn't work. 

But seeing the [Pimcore documentation](https://pimcore.com/docs/platform/Ecommerce_Framework/Index_Service/Product_Index_Configuration/Elastic_Search/Configuration_Details), the Elasticsearch configuration is linked to the E-commerce Bundle. 

My project doesn't need it, but to make it work, do I need the e-commerce bundle?

**Comment by NiklasBr** [comment]
You do not need the Ecom framework bundle, only `pimcore/elasticsearch-client` which is [already required in the composer.json on this package](https://github.com/BrandOriented/PimcoreCiHubConnector/blob/master/composer.json#L43). Follow [the elasticsearch-client installation instructions](https://github.com/pimcore/elasticsearch-client) and then the ones for CI Hub, then it ought to work.

**Reply by designjpp** [reply]
thanks, 

I will try again
